x-pack/plugins/observability_solution/apm/public/components/app/error_group_details/error_sampler/error_sample_detail.tsx

```diff
@@ -352,8 +352,8 @@ export function ErrorSampleDetailTabContent({
   currentTab: ErrorTab;
 }) {
   const codeLanguage = error?.service.language?.name;
-  const exceptions = error?.error.exception || [];
-  const logStackframes = error?.error.log?.stacktrace;
+  const exceptions = error?.error?.exception || [];
```

Comment:
the first `?` is not necessary - `APMError` is always defined (I understand these were here before, but let's not add to it)